src/components/top-bar/_top-bar.scss

```diff
@@ -50,7 +52,7 @@
   .right-controls {
     float: right;
     text-align: left;
-    margin: #{$top-bar-top-bottom-gutter}px #{$gui-gutter}px #{$top-bar-top-bottom-gutter}px 0;
+    margin: #{$top-bar-top-bottom-gutter}px #{$gui-gutter}px #{$top-bar-top-bottom-gutter}px 32px;
```

Comment:
what does this margin change do ?

Reply:
it adds some space to the left-most edge of the right-controls - it spaces between left and right controls when they collide. so bottom line it makes it look better.